2016/6xxx/CVE-2016-6555.json

```diff
@@ -1,18 +1,99 @@
 {
     "CVE_data_meta": {
-        "ASSIGNER": "cve@mitre.org",
+        "ASSIGNER": "cve@rapid7.com",
+        "DATE_PUBLIC": "2016-09-14T18:18:00.000Z",
         "ID": "CVE-2016-6555",
-        "STATE": "RESERVED"
+        "STATE": "PUBLIC",
+        "TITLE": "OpenNMS Stored XSS via SNMP Trap Alerts"
     },
+    "affects": {
+        "vendor": {
+            "vendor_data": [
+                {
+                    "product": {
+                        "product_data": [
+                            {
+                                "product_name": "OpenNMS",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_affected": "<=",
+                                            "version_name": "18.0.1",
+                                            "version_value": "18.0.1"
+                                        }
+                                    ]
+                                }
+                            }
+                        ]
+                    },
+                    "vendor_name": "OpenNMS"
+                }
+            ]
+        }
+    },
+    "credit": [
+        {
+            "lang": "eng",
+            "value": "This issue was discovered by Matthew Kienow and disclosure was coordinated by Rapid7."
+        }
+    ],
     "data_format": "MITRE",
     "data_type": "CVE",
     "data_version": "4.0",
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "OpenNMS version 18.0.1 and prior are vulnerable to a stored XSS issue due to insufficient filtering of SNMP trap supplied data. By creating a malicious SNMP trap, an attacker can store an XSS payload which will trigger when a user of the web UI views the data. This issue was fixed in version 18.0.2, released on September 20, 2016."
```

Comment:
If you want to be specific replace "views the data" with "views the events list page".

Reply:
"18.0.2-1" to match the other CVE description

Reply:
Yeah meant to change it here, the final release was just 18.0.2